gendiff/scripts/gendiff.py

```diff
@@ -16,7 +45,11 @@ def main():
         help="set format of output",
     )
     args = parser.parse_args()
-    print(parser, args)
+
+    if args == "-h" or args == "--help":
```

Comment:
`argparse` should be able to handle such cases without your help, consider removing this `if/else` block